critiquebrainz/db/moderation_log.py

```diff
@@ -0,0 +1,139 @@
+"""
+Methods here define logs for various activities that the moderators can take
+via the moderator interface. A new log entry is created for every action.
+"""
+from critiquebrainz.data import db
+import sqlalchemy
+from datetime import datetime
+
+
+ACTION_HIDE_REVIEW = "hide_review"
+ACTION_BLOCK_USER = "block_user"
+
+def create(*, admin_id, review_id=None, user_id=None,
+           action, reason):
+    """Log blocking a user or hiding a review with IDs.
+
+    Args:
+        admin_id (uuid): ID of the admin.
+        user_id (uuid): ID of the user blocked.
+        review_id (uuid): ID of the review hidden.
+        action (str): str("hide_review", "block_user").
+        reason (str): Reason for blocking a user or hiding a review.
+    """
+    if not review_id and not user_id:
+        raise TypeError("No review ID or user ID specified.")
```

Comment:
[`ValueError`](https://docs.python.org/3.6/library/exceptions.html#ValueError) would be more appropriate here and below.